pkgs/servers/nosql/mongodb/mongodb.nix

```diff
@@ -1,27 +1,50 @@
-{ lib, stdenv, fetchurl, sconsPackages, boost, gperftools, pcre-cpp, snappy, zlib, libyamlcpp
-, sasl, openssl, libpcap, curl, Security, CoreFoundation, cctools }:
+{ lib
+, stdenv
+, fetchurl
+, sconsPackages
+, boost
+, gperftools
+, pcre-cpp
+, snappy
+, zlib
+, libyamlcpp
+, sasl
+, openssl
+, libpcap
+, python3
+, curl
+, Security
+, CoreFoundation
+, cctools
+, lzma ? null
```

Comment:
```suggestion
, lzma
```

Reply:
Fails basic evaluation test if I skip the `null`

Reply:
This means there is something wrong here. lzma is an alias to xz. Either replace lzma with xz or remove it entirely if xz is already part of the inputs/buildInputs.

Reply:
Not quite. The lzma/xz dependency is only needed starting in MongoDB 4.4. on x86_64-linux. For earlier versions it is not required thus the option. Is there a more elegant way to do this?

Reply:
I believe you can bring it to scope but add it as input of the versions that is newer than 4.4.